atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -207,6 +207,9 @@ public class SnapshotTransaction extends AbstractTransaction
     private static final int BATCH_SIZE_GET_FIRST_PAGE = 1000;
     private static final long TXN_LENGTH_THRESHOLD = Duration.ofMinutes(30).toMillis();
 
+    @VisibleForTesting
+    static final int MAX_POST_FILTERING_ITERATIONS = 200;
```

Comment:
This was determined by simply counting the number of calls to `getWithPostFilteringIterate` via the stacktrace; it was only 173.